ACE/ace/OS_NS_signal.inl

```diff
@@ -85,7 +86,11 @@ ACE_INLINE int
 sigaddset (sigset_t *s, int signum)
 {
   ACE_OS_TRACE ("ACE_OS::sigaddset");
-#if defined (ACE_LACKS_SIGSET)
+#if defined (ACE_LACKS_SIGADDSET)
```

Comment:
config-mqx.h already sets this `ACE_LACKS_SIGADDSET` which isn't used in the code until now, something to check

Reply:
Looks like it is added to config-mqx.h in PR #917 but isn't used anywhere else. I don't have an MQX system to test, so I don't know whether we should keep it in config-mqx.h.

Reply:
Safest seems to remove ACE_LACKS_SIGADDSET from config-mqx.h, at the moment it is required a compile error is given, if you keep it there and it shouldn't be there it could give only a strange runtime issue